README.rst

```diff
@@ -74,9 +79,13 @@ Internationalization & Translation
 
 This application has internationalization and translation enabled.
 
-- If you create any new translatable content, you should run `makemessages <https://docs.djangoproject.com/en/3.1/ref/django-admin/#makemessages>`_ to create or update message files.
+- If you create any new translatable content, you should run `makemessages <https://docs.djangoproject.com/en/3.1/ref/django-admin/#makemessages>`_ to create or update message files. We use a customized version of this command, available in ``/geniza/common/management/commands/makemessages.py``.
+
+	django-admin makemessages
+
+- Before running the app, you should run `compilemessages <https://docs.djangoproject.com/en/3.1/ref/django-admin/#compilemessages>`_ to generate compiled translations.
```

Comment:
Could you add this to `DEPLOYNOTES.md`?

Reply:
I think we usually use `DEPLOYNOTES` for one-off tasks that need to run after a particular version is deployed, rather than tasks that should run every deploy. I'm working on a `django` role for our `cdh-ansible` repo that will run this command on every deploy (among other things): https://github.com/Princeton-CDH/cdh-ansible/pull/73